### PR TITLE
fix #428

### LINKED ISSUE #428: Incomplete Italian translation
The XIB files are missing Italian translations.

### PR DESCRIPTION
**Pull request by hpaolini**
added missing french string and italian translation

### CODE REVIEW

File: Spectacle/Resources/Localizations/fr.lproj/Spectacle.strings [hunk 1]
```diff
@@ -88,6 +88,11 @@
 
 /* --- */
 
+/* Class = "NSMenuItem"; title = "Restore Defaults..."; ObjectID = "aFM-wq-kOy"; */
+"aFM-wq-kOy.title" = "Rétablir par défaut...";
```
Comment (eczarny) [inline Spectacle/Resources/Localizations/fr.lproj/Spectacle.strings]: I don't think this is necessary. This menu item has been removed in favor of the button in the preferences window.

File: Spectacle/Resources/Localizations/it.lproj/Spectacle.strings [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
 
 /* Class = "NSMenuItem"; title = "Redo"; ObjectID = "187-7k-0Ct"; */
-"187-7k-0Ct.title" = "Redo";
+"187-7k-0Ct.title" = "Ripristina";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Restore Defaults..."; ObjectID = "aFM-wq-kOy"; */
-"aFM-wq-kOy.title" = "Restore Defaults...";
+"aFM-wq-kOy.title" = "Ripristina predefiniti...";
```
Comment (eczarny) [inline Spectacle/Resources/Localizations/it.lproj/Spectacle.strings]: Same here. We probably don't need this.

File: Spectacle/Resources/Localizations/it.lproj/Spectacle.strings [hunk 3]
```diff
@@ -1,111 +1,111 @@
...
+"l9O-0B-fjp.title" = "Annulla";
 
 /* Class = "NSMenuItem"; title = "Redo"; ObjectID = "187-7k-0Ct"; */
-"187-7k-0Ct.title" = "Redo";
+"187-7k-0Ct.title" = "Ripristina";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Restore Defaults..."; ObjectID = "aFM-wq-kOy"; */
-"aFM-wq-kOy.title" = "Restore Defaults...";
```
Comment (eczarny) [inline Spectacle/Resources/Localizations/it.lproj/Spectacle.strings]: You can probably delete this too. Odds are I missed removing this when I originally created these files.

**Comment by eczarny** [comment]
This is so awesome, thank you for the contribution! Just a few nits before merging.

File: Spectacle/Resources/Localizations/it.lproj/Spectacle.strings (hunk 3 above)
Comment (hpaolini) [inline Spectacle/Resources/Localizations/it.lproj/Spectacle.strings]: Yes, this line actually was replaced by the Italian translation. Gone anyways.

**Comment by hpaolini** [comment]
You're welcome! Please check the changes.

**Pull request merged**